recipes/uropa/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Main program
+mv uropa.py $PREFIX/bin
+
+mkdir -p $PREFIX/bin/uropa
```

Comment:
Can this be located under lib?

Reply:
I honestly don't know. Fact is that `uropa.py` relies on this (relative) structure to include further source files located in `uropa/` via pythons `include uropa.config`. Any idea how this could be solved? Creating a proper Pypi source package that can be installed via `python setup.py install`?

Reply:
From a community standpoint a pypi package would be ideal, but I will not push you in this direction. The PR is ok for BioConda as it is I think. But consider a real pypi package.